README.md

```diff
@@ -33,32 +33,19 @@ $$(
   subtractPercent(20),
   // add tax
   addPercent(10)
-).$; // 88
+).toNumber(); // 88
 ```
 
 ### Known Issues
 
-This code was written in ES6, and no attempt has been made to compile it for older browsers. This should not be a problem in any modern evergreen browser, but it will likely throw errors in old IE and old versions of Safari. You have been warned.
-
-This code scales money calculations to operate on cents automatically, but fractions of cents are still represented in IEEE 754 floating point, which means that `$.of(.1) + $.of(.2) !== $.of(.3).valueOf()`. IEEE 754 can't accurately represent the value, so it rounds to `0.30000000000000004`.
-
-Values are stored with full floating point precision to give you more headroom for complex financial calculations which require more decimal digits.
-
-That's intentional. By preserving the full floating point precision, you get roughly 16 digits of fractional cent precision for headroom, which minimizes the impact of cumulative rounding errors for more complex calculations.
-
-When you use `$.of()` to directly lift cents into the money type, and `money.valueOf()` (or any of the native JS math operators) to get values out, you get the full range of precision, unrounded.
-
-Usually, you will want those values rounded to the nearest cent, so when you use the convenient `$` and `cent` lifts and getters, rounding errors get rounded away:
+This code was written in ES6, and no attempt has been made to compile it for older browsers. This should not be a problem in any modern evergreen browser, but it will likely throw errors in old IE and old versions of Safari. You have been warned. If you want to run the code in other browsers, you'll need to compile to ES5 yourself.
 
-```
-$(.001) + $(.002) === $(.003).cents
-```
+Values are stored in arbitrary precision using BigNumber, so you can perform accurate calculations for cryptocurrencies such as Bitcoin or Ethereum which have 8 and 18 decimal precision, respectively. By way of contrast, JavaScript's native number type is IEEE 754 with 16 digits of decimal precision.
 
 To recap:
 
-* By default, all math operations automatically use the full range of precision using `$.of()` and `money.valueOf()`.
-* If you want to round to the nearest cent when you lift the initial value into the type, use `$(x)` or `$.cents(x)`.
-* If you want to round the value to the nearest cent to display to users or charge accounts, use the `money.$` or `money.cents` getters.
+* By default, all math operations automatically use arbitrary precision big numbers internally.
+* You can get the value in a number type using `.toNumber()`.
 
 
 ## Getting Started
```

Comment:
I don't know how to add a review to code that didn't change 😅 

But in the docs, we still have the imports declared as `import { $ } from 'moneysafe';` and `const { $ } = require('moneysafe');`. Money$afe doesn't export `$`, but `ethereum`. We should either export `ethereum` also as `$` or change the import examples here in the docs (lines 62 and 69).